File: foundations/py/5/fibonacci.py
n = int(input('Please enter a positive integer greater than 1:'))

p1 = 1
print(p1)
p2 = 1
print(p2)

for i in range(n-2):
    p2 += p1
    p1 = p2 - p1
    print(p2)

File: foundations/cpp/6/fibonacci.cpp
#include <iostream>
using namespace std;

int main()
{
	int n;
	cout << "Please enter a positive integer greater than 1: ";
	cin >> n;

	int prior = 0, current = 1;
	do
	{
		cout << current << endl;
		current += prior;
		prior = current - prior;
	} while (--n > 0);

	return 0;
}